source/agora/consensus/EnrollmentManager.d

```diff
@@ -389,7 +390,7 @@ public class EnrollmentManager
     public bool hasEnrollment (const ref Hash enroll_hash) @trusted
     {
         auto results = this.db.execute("SELECT EXISTS(SELECT 1 FROM validator_set " ~
-            "WHERE key = ?)", enroll_hash[]);
+            "WHERE key = ?)", reverseUbytes(enroll_hash));
```

Comment:
Ultimately, the library should accept either ranges, or recognize `toString`. Could you check what code is taken here ?

Reply:
Ping @TrustHenry

Reply:
This is a library for use only here.
If we publicize this, better add a function in Hash.
Am I right?

Reply:
By "the library" I mean d2sqlite.

Reply:
Oh, I will check.